src/components/BountyCard/BountyCard.module.scss

```diff
@@ -1,50 +1,57 @@
 @import "~styles/variables.scss";
 
 .bountyCardContainer {
+  border-left: 3px solid transparent;
```

Comment:
Should these also scale with screen size?

Reply:
No this should remain static. This and things like border-radii on containers are things we don't want fluctuating.